src/runtime_src/doc/toc/yocto.rst

```diff
@@ -22,9 +18,61 @@ Before start to build Linux image, make sure your have:
 The PetaLinux tool chain can be downloaded from xilinx.com.
 If you don't have a .hdf file, please see :ref:`Build Boot Images`.
 
+*NOTE* The purpose of this page is only for providing general getting started guidance.
+If you are interested in details about creating software images for embedded platform.
+Please read the Software Platform section of Xilinx® Document UG1146.
+
 Create PetaLinux Project with XRT recipes
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+**NOTE**: Beginning in Xilinx release 2019.2, PetaLinux contains recipes for XRT
+and all required software prerequisites. This information is provided as reference
+for those who wish to build within a standard Yocto flow *or* for those who wish
+to customize or extend these recipes. For most use cases, *do not* add the recipes
+from the XRT repository into a 2019.2+ PetaLinux project and instead enable
+XRT through the root file system menuconfig utility.
+
+.. code-block:: bash
+
+        #Configure Linux kernel (default kernel config is good for zocl driver)
+        $ petalinux-config -c kernel
+
+        # Configure rootfs, enable below components:
+        #   menu -> "Filesystem Pacakges" -> "libs" -> xrt
+        #   menu -> "Filesystem Pacakges" -> "libs" -> xrt-dev
+        $ petalinux-config -c rootfs
+
+Add XRT kernel node in device tree
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+An example of device tree is in ``<XRT>/src/runtime_src/core/edge/fragments/xlnk_dts_fragment_mpsoc.dts``. You can attach it to ``project-spec/meta-user/recipes-bsp/device-tree/files/system-user.dtsi`` in your PetaLinux project.
```

Comment:
The fragments folder was removed. Should we point user to Vitis_Embedded_Platform_Source repo as reference?